tests/ui_test/homepage.spec.ts

```diff
@@ -3,4 +3,41 @@ import { test, expect } from '@playwright/test';
 test('Verify homepage URL', async ({ page }) => {
```

Comment:
If you verify not only url, but the whole homepage with this test - would you mind to change the test name to Verify homepage url and content?